app/templates/components/button-activate-all-github.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if this.isVisible}}
+  <button
+    class="button--blue"
+    data-test-activate-button
+    {{on 'click' (action 'activateAll')}}
+  >
+    <SvgImage @name={{vcs-icon this.owner.vcsType}} @class="fill-current align-text-bottom w-5 h-5 ml-1 mr-2" />
```

Comment:
I think this is very GitHub specific logic and won't work with other VCSes, so the icon image could be just github.

Reply:
Yeah, that makes sense to me. Change made :)